addons/t9n/security/ir.model.access.csv

```diff
@@ -1 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+t9n.access_t9n_project,access_t9n_project,t9n.model_t9n_project,base.group_system,1,1,1,1
```

Comment:
You should mention the group you're granting access to in the name and ID of the rule, otherwise another rule for a different group would have the same name
```suggestion
access_t9n_project_system,t9n.project.system,model_t9n_project,base.group_system,1,1,1,1
```